content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -39,6 +39,20 @@ If you have a starting point in mind, but aren't sure what your users did next,
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_dropoffs.png" alt="The funnel step editor automatically loads the top most common views and actions that users typically see and take next." style="width:50%;" >}}
 
+<div class="alert alert-info"> Any action or view that happens between two steps in a funnel does not impact the step-by-step or overall conversion rate. As long as step 1 and step 2 happen in the right order in a given session at least once, it counts as a single converted session.</div>
```

Comment:
```suggestion
<div class="alert alert-info"> Any action or view between two steps in a funnel does not impact the step-by-step or overall conversion rate. As long as step 1 and step 2 happen in the right order in a given session at least once, it counts as a single converted session.</div>
```

Reply:
Now that I think about it, maybe this would make more sense in the section about conversion computing?